faltest/smoke-test.js

```diff
@@ -1,25 +1,54 @@
 'use strict';
 
 const { setUpWebDriver } = require('@faltest/lifecycle');
-const Server = require('ember-cli-test-server');
+const { promisify } = require('util');
+const request = promisify(require('request'));
+const execa = require('execa');
 const assert = require('assert');
 const { percySnapshot } = require('@percy/webdriverio');
 
 describe('smoke', function() {
   setUpWebDriver.call(this);
 
   before(async function() {
-    this.server = new Server();
-
-    this.port = await this.server.start();
+    this.url = await new Promise(resolve => {
+      (async function getUrl() {
+        let commit;
+        if (process.env.TRAVIS_PULL_REQUEST_SHA) {
+          commit = process.env.TRAVIS_PULL_REQUEST_SHA;
+        } else {
+          commit = (await execa.command('git rev-parse HEAD')).stdout;
+        }
+
+        let repo = 'ember-vr/ember-aframe-shim'
+
+        let { body } = await request({
+          url: `https://api.github.com/repos/${repo}/statuses/${commit}`,
+          headers: {
+            'User-Agent': repo
+          },
+          json: true
+        });
+
+        let status = body.find(status => status.context === 'netlify/ember-aframe-shim/deploy-preview');
+
+        let fallback = 'https://ember-aframe-shim.netlify.com/';
+
+        if (!status) {
+          resolve(fallback);
+        }
+
+        if (status.state !== 'pending') {
+          resolve(status.target_url);
```

Comment:
is it possible this isn't set before the preview is deployed?

Reply:
yep, that's why it's gated behind !pending.